src/PageActionButton.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import Button from './Button';
+import Icon from './Icon';
+import Flex from './Flex';
+import FlexItem from './FlexItem';
+
+export const PAGE_ACTION_BUTTON_CLASS = 'pageActionButton';
+
+/**
+ * @module PageActionButton
+ */
+class PageActionButton extends React.Component {
+	render() {
+		const {
+			children,
+			className,
+			icon,
+			label,
+			stackedIcon,
+			short,
+			onClick,
+			...other
+		} = this.props;
+
+		let direction,
+			switchDirection,
+			justify;
+
+		// if column
+		// icons and labels in rows,
+		// adjust to take up space based on position and numbers
+		//
+		//  OTHERWISE:
+		// always a nice column of actions
+		// each action is a horizontal icon and label
```

Comment:
Sorry to be particular here but it's unclear what exactly this comment mean. Can you clarify this a bit?